owncloudApp/src/main/res/values/colors.xml

```diff
@@ -69,6 +69,5 @@
     <color name="bottom_sheet_fragment_title_color">#6E758C</color>
 
     <!-- Spaces -->
-    <color name="spaces_card_background_color">#edf3fa</color>
     <color name="spaces_card_default_image_color">#4c5f79</color>
```

Comment:
Should it be brandable too? Maybe we could reuse another color for this divider..

Reply:
Ok, I reused another similar one. Anyway, in a beginning this was for the tint of the default icon when the space had no image, but now we print it in black so...